README.md

```diff
@@ -1,2 +1,63 @@
+[![Codacy - Quality](https://img.shields.io/codacy/grade/39d312bf98244961975559f141c3e000?logo=Codacy)](https://www.codacy.com/manual/edaa-org/pySystemVerilogModel)
+[![Codacy - Coverage](https://img.shields.io/codacy/coverage/39d312bf98244961975559f141c3e000?logo=Codacy)](https://www.codacy.com/manual/edaa-org/pySystemVerilogModel)
+[![Codecov - Branch Coverage](https://img.shields.io/codecov/c/github/edaa-org/pySystemVerilogModel?logo=Codecov)](https://codecov.io/gh/edaa-org/pySystemVerilogModel)
+[![Libraries.io SourceRank](https://img.shields.io/librariesio/sourcerank/pypi/pySystemVerilogModel)](https://libraries.io/github/edaa-org/pySystemVerilogModel/sourcerank)  
+[![GitHub Workflow Release Status](https://img.shields.io/github/workflow/status/edaa-org/pySystemVerilogModel/Release?label=release&logo=GitHub%20Actions&logoColor=FFFFFF)](https://github.com/edaa-org/pySystemVerilogModel/actions?query=workflow%3A%22Release%22)
+[![PyPI](https://img.shields.io/pypi/v/pySystemVerilogModel?logo=PyPI&logoColor=FBE072)](https://pypi.org/project/pySystemVerilogModel/)
+![PyPI - Status](https://img.shields.io/pypi/status/pySystemVerilogModel?logo=PyPI&logoColor=FBE072)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pySystemVerilogModel?logo=PyPI&logoColor=FBE072)
+[![Libraries.io status for latest release](https://img.shields.io/librariesio/release/pypi/pySystemVerilogModel)](https://libraries.io/github/edaa-org/pySystemVerilogModel)
+[![Requires.io](https://img.shields.io/requires/github/edaa-org/pySystemVerilogModel)](https://requires.io/github/EDAA-ORG/pySystemVerilogModel/requirements/?branch=main)  
+[![GitHub Workflow - Documentation Status](https://img.shields.io/github/workflow/status/edaa-org/pySystemVerilogModel/Documentation?label=documentation&logo=GitHub%20Actions&logoColor=FFFFFF)](https://github.com/edaa-org/pySystemVerilogModel/actions?query=workflow%3A%22Documentation%22)
+[![Documentation License](https://img.shields.io/badge/doc%20license-CC--BY%204.0-green)](LICENSE.md)
+[![Documentation - Read Now!](https://img.shields.io/badge/doc-read%20now%20%E2%9E%94-blueviolet)](https://edaa-org.github.io/pySystemVerilogModel/)
+
 # pySystemVerilogModel
-An abstract language model of SystemVerilog (incl. verilog) written in Python.
+
+An abstract language model of SystemVerilog (incl. Verilog) written in Python.
+
+## Main Goals
+This package provides a unified abstract language model for SystemVerilog (incl.
+Verilog). Projects reading from source files can derive own classes and implement
+additional logic to create a concrete language model for their tools.
+
+Projects consuming pre-processed VHDL data (parsed, analyzed or elaborated) can
+build higher level features and services on such a model, while supporting multiple
+frontends.
+
+
+## Use Cases
+**pySystemVerilogModel Generators**
+* TBD
+
+**pySystemVerilogModel Consumers**
+* TBD
+
+
+## Examples
+### List all Modules with Parameters and Ports
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Above](https://app.codacy.com/gh/edaa-org/pySystemVerilogModel/pullRequest?prid=8142225)
```suggestion

### List all Modules with Parameters and Ports
```